rocprofiler-systems/Jacobi/README.md

```diff
@@ -0,0 +1,228 @@
+# Rocprof-sys
+NOTE: extensive documentation on how to use rocprof-sys for the [GhostExchange examples](https://github.com/amd/HPCTrainingExamples/tree/main/MPI-examples/GhostExchange) is also available as `README.md` in this exercises repo. Here, we show how to use rocprof-sys tools considering the example in HPCTrainingExamples/HIP/jacobi.
+
+In this series of examples, we will demonstrate profiling with rocprof-sys on a platform using an AMD Instinct&trade; MI300 GPU, hosted in AAC environment. ROCm releases (6.3.2) include rocprof-sys.
+
+Note that the focus of this exercise is on rocprof-sys profiler, not on how to achieve optimal performance on MI300A.
+
+First, start by cloning HPCTrainingExamples repository and loading ROCm:
+
+```
+git clone https://github.com/amd/HPCTrainingExamples.git
+```
+
+## Environment setup
+
+For this training, one requires recent ROCm (>=6.3) which contains rocprof-sys, as well as an MPI module.
```

Comment:
Instead of saying "an MPI module", I would say, "an MPI installation". This would cover systems that don't use modules.